### ISSUE TITLE
#a11y: Lack of clearly visible focus indication for some interface components

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/desktop#1332]
### Steps To Reproduce

1. Open Bitwarden
2. Use `Tab` / `Shift+Tab` to navigate using the keyboard

### Expected Result

A visible keyboard focus indication on the "My Vault" and "Send" controls.

### Actual Result

While there does appear to be a subtle shift in background colour, there's no clear indication which one has focus (compare with other elements that have a reasonably clear indicator)

### Screenshots or Videos

Animated GIF of navigating through the application - note that after the "No folder" focus goes to "My Vault", then "Send", but there really isn't much of a visual indication.
![bitwarden-desktop-focus-indication](https://user-images.githubusercontent.com/895831/153774875-0ea38f4d-b564-4e1e-a069-ed762ff9ac76.gif)

### Operating System

Windows

### Operating System Version

10

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 1.31.2 Shell 16.0.7 Renderer 96.0.4664.110 Node 16.9.1 Architecture x64

**Comment by Hinton** [comment]
Hi @patrickhlauke,

Thanks for reporting this. If I understand it correctly it's primarily the `My Vault` and `Send` tabs in the bottom left corner that are missing focus styling? The `Test Identity` seems to also lack the top border as it's obstructed by the search bar.

**Comment by patrickhlauke** [comment]
yes, primarily the `My Vault` and `Send` since they pretty much lack any discernible/clear focus indication. the `Test Identity` (and well, all items really) could do with a proper full outline (not just the little sliver at the top) if feasible, but at least they arguably have something visible already there (with the subtle change in colour and the left-hand border changing)

**Comment by squarepupil** [comment]
I will also note that I only discovered this issue because the UI behavior recently changed when hitting tab after entering a search query. It used to jump the cursor to the list of search results and now (frustratingly) jumps the cursor to the account selection dropdown followed by the menu on the left side of the application, which can be seen in the gif at the top of this issue report.

As a by-the-way, does anyone know how to quickly get to your first search result without having to touch the mouse?

**Comment by patrickhlauke** [comment]
@squarepupil this will likely be one of those "squaring the circle" type problems. for some folks the old focus order will make sense, for others the current one. i think the only way around that would be to start treating the different areas/panels in the app as separate regions that you jump between using something like `F6` (standard at least on Windows, can't remember offhand if macOS and Linux have a similar key)

**Comment by patrickhlauke** [comment]
x-ref https://github.com/bitwarden/desktop/issues/1325
